compiler/src/dotty/tools/dotc/config/CliCommand.scala

```diff
@@ -123,15 +124,30 @@ trait CliCommand:
     prefix + "\n" + availableOptionsMsg(cond)
 
   protected def isStandard(s: Setting[?])(using settings: ConcreteSettings)(using SettingsState): Boolean =
-    !isAdvanced(s) && !isPrivate(s)
+    !isVerbose(s) && !isWarning(s) && !isAdvanced(s) && !isPrivate(s) || s.name == "-Werror" || s.name == "-Wconf"
+  protected def isVerbose(s: Setting[?])(using settings: ConcreteSettings)(using SettingsState): Boolean =
+    s.name.startsWith("-V") && s.name != "-V"
+  protected def isWarning(s: Setting[?])(using settings: ConcreteSettings)(using SettingsState): Boolean =
+    s.name.startsWith("-W") && s.name != "-W" || s.name == "-Xlint"
```

Comment:
There is only one option of a kind `-V` and `-W` each added in this PR. Is it worth creating these two categories? What's the motivation?

Reply:
Lets a thousand flowers bloom and still be manageable. https://github.com/lampepfl/dotty/issues/12785

If I want more output merely (not change in behavior), `-V` is more succinct than checking `-X` and `-Y` (which become a long list eventually). Everyone needs `-Vprint` at some point, not just compiler coders.

Reply:
Another motivation is alignment with Scala 2.